__tests__/app.test.js

```diff
@@ -63,3 +63,75 @@ describe.only("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET : /api/articles object including total comments", () => {
+  test("/api/articlezz -> route that does not exist: 404 Not Found", () => {
+    return request(app)
+      .get("/api/articlezz")
+      .expect(404)
+      .then(({ body: { message } }) => {
+        expect(message).toBe("not found");
+      });
+  });
```

Comment:
this is a good test but be aware that you only need one 'route-not-found' test!

no need to write any more after this one 👍